### ISSUE TITLE
Token Exchange Fails Silently

### ISSUE THREAD

**Issue by cpendery** [microsoft/BotFramework-WebChat#4735]
### Is it an issue related to Adaptive Cards?

No

### Is this an accessibility issue?

No

### What version of Web Chat are you using?

Latest production

### Which distribution are you using Web Chat from?

NPM

### Which hosting environment does this issue primarily affect?

Web apps

### Which browsers and platforms do the issue happened?

Browser: Edge (latest), Browser: Chrome (latest), Browser: Firefox (latest)

### Which area does this issue affect?

Attachment: OAuth card

### Please describe the bug

When trying to trigger a token exchange in React via the `usePostActivity` hook as shown below, if the token exchange is unsuccessful, I am unable to detect this failure. `postActitivity` does not throw an exception & there is no failure item in the `useActivity` hook. 
```javascript
postActivity({
    channelData: { invokeId },
    type: 'invoke',
    name: 'signin/tokenExchange',
    value: {
      id,
      connectionName,
      token
    }
  })
```

### How to reproduce the issue?

1. Trigger the invoke action in React with an invalid OAuth Token Exchange configuration

### What do you expect?

I'd expect to be able to detect that an error has occurred. 

### What actually happened?

I see a failed network request (502), but that failure isn't populated in a way where I can detect it

**Comment by stevkan** [comment]
@cpendery - I'll start looking into this to see if I can repro. That being said, are you able to provide a copy of your web chat code or a minimum reproducible example (minus any secrets/keys)?

**Comment by cpendery** [comment]
> @cpendery - I'll start looking into this to see if I can repro. That being said, are you able to provide a copy of your web chat code or a minimum reproducible example (minus any secrets/keys)?

I'll make an example and ping it here

**Comment by cpendery** [comment]
@stevkan Here is an example of how I'm making the call that silently fails: https://github.com/cpendery/bot-framework-sso-failure-example

**Comment by stevkan** [comment]
@cpendery - Apologies for the extended delay. This fell off my radar and I only recently found it, again.

I don't know if you are still having issues or not. However, I downloaded a copy of your project you linked above and was able to both generate a token and start a conversation with a bot.

I did make a couple minor changes by commenting out the login button from the exported App and setting `showChat` to `true` so Web Chat would just start up for me. And, of course, I supplied my DirectLine secret in the Authorization header.

Outside of those changes, it runs as expected. If you are still having an issue, try removing the same components to be sure the login feature isn't somehow interfering with the rest of the app.
